### ISSUE TITLE
pyspark --packages databricks:tensorframes:0.2.3-s_2.10 not work

### ISSUE THREAD

**Issue by jq** [databricks/tensorframes#48]
seems not yet publish

**Comment by thunterdb** [comment]
Yes, an issue with the spark-package website is preventing the new release from being uploaded. I am working with the maintainer of spark-packages to solve this issue.

**Comment by sergulaydore** [comment]
Did you solve this issue? The following script does not work:

`os.environ['PYSPARK_SUBMIT_ARGS'] = '--packages com.databricks:tensorframes:0.2.3-s_2.10 pyspark-shell'

import pyspark
import tensorflow as tf
import tensorframes as tfs
`
Can you please let me know what I am doing wrong?

Thanks,
Sergul
